config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -99,6 +99,56 @@ periodics:
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-kubelet-serial-containerd-flaky
```

Comment:
We may want to remove `serial` from the name as we removed `[Serial]` from the focus.

Reply:
good idea.

Reply:
updated